dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -62,9 +62,18 @@ def names
     render json: user_ml_model_data.to_json
   end
 
+  # Model ids are 12-character random alphanumeric strings.
+  def valid_model_id?(id)
```

Comment:
i think this method should live on the `UserMlModel` model. it would need to be a static/class method since we don't have an instance, but this should be a concern of the model itself

Reply:
moved!